src/iodide-api/resolve.js

```diff
@@ -0,0 +1,23 @@
+
+window.iodideRequireExplicitResolution = false
+window.iodideRequireExplicitResolutionStatus = undefined
+
+const expectResolution = () => {
+  window.iodideRequireExplicitResolution = true
+  window.iodideRequireExplicitResolutionStatus = 'pending'
+  window.iodideExplicitResolver = new Promise((resolve) => {
+    // poll for resolution, given there is no way to do this strictly w/ Promises
+    const interval = setInterval(() => {
+      if (window.iodideRequireExplicitResolutionStatus === 'resolved') {
+        resolve()
+        clearInterval(interval)
+      }
+    }, 100)
+  })
+}
+
+const resolve = () => { console.log('RESOLUTION TIME'); window.iodideRequireExplicitResolutionStatus = 'resolved' }
```

Comment:
you can just `export const resolve` rather than needing the export block at the bottom (i think we should adopt this practice throughout the code base)